Remotion/Web/Core/Web.Core.csproj

```diff
@@ -32,7 +32,7 @@
     <Reference Include="System.Configuration" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="..\Development.Analyzers\bin\$(Configuration)\netstandard2.0\Remotion.Web.Development.Analyzers.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <ProjectReference Include="..\Development.Analyzers\Web.Development.Analyzers.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" SetTargetFramework="targetframework=netstandard2.0" />
```

Comment:
This is different to Core.Core. please re-check.

Reply:
Thanks for spotting my missclicks!